metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
I don't think I made my point clearly enough.  I don't have a problem with having the ability to add enrichments, indexing definitions, etc in an extension.  

My point is that the defaults we provide should make as few assumptions about the environment as possible. 
 For example, when I install the Bro parser extension from this PR, it is going to perform GeoIP and malicious IP lookups.  That assumes that the environment in which I am installing this has the Geo database setup correctly and a malicious IP table setup in HBase.  

Are those fair assumptions to make?  It seems to me we should make as few assumptions as possible so this can "just work" for the greatest number of users out there.  I would default to no enrichments, for example.